### PR TITLE
Add `-100` versions of colours as background color classes.

### LINKED ISSUE #809: [@thumbtack/thumbprint-atomic]: Add 100-level background colors to Atomic
@jtabach requested that we add Blue 100 as a background color. I looked it up and we use it 20+ times as a background color in our SCSS. This passes our threshold for adding a new class: https://thumbprint.design/atomic/usage/#section-adding-new-atomic-classes

We should add the class to Atomic (as well as all of the other 100-level background classes for consistency).

### PR DESCRIPTION
**Pull request by lavelle**
Closes #809

### CODE REVIEW

File: packages/thumbprint-atomic/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Added
+
+-   [Minor] Add `-100` versions of colours as background color classes. ([#809](https://github.com/thumbtack/thumbprint/issues/809))
```
Comment (danoc) [inline packages/thumbprint-atomic/CHANGELOG.md:7]:
```suggestion
-   [Minor] Add `-100` versions of colours as background color classes. (#809)
```

Let's do just the issue number for consistency with other entries. Although I agree that having the link is useful. Maybe that can be a different PR.

**Review by danoc — COMMENTED** [review]
The snapshot still needs updating, but LGTM otherwise.

File: packages/thumbprint-atomic/CHANGELOG.md (hunk 1 above)
Comment (lavelle) [reply packages/thumbprint-atomic/CHANGELOG.md:7]: Fair enough. This is actually a feature of GitLens, it autosuggested this link when i typed the issue number 🤯

**Review by danoc — APPROVED** [review]
